src/lib/logSummary/logSummary.cpp

```diff
@@ -0,0 +1,164 @@
+/*
+*
+* Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Ken Zangelin
+*/
+#include <unistd.h>
+#include <semaphore.h>
+#include <errno.h>
+#include <pthread.h>
+
+#include "logMsg/logMsg.h"
+#include "logMsg/traceLevels.h"
+
+#include "alarmMgr/alarmMgr.h"
+#include "logSummary/logSummary.h"
+
+
+
+/* ****************************************************************************
+*
+* Global vars
+*/
+static sem_t               lsSem;
```

Comment:
Nope.
This is a global variable, the other one is a member of the class.
There is no reason to prefix a class member.
Actually as *lsSem* is static, there is no reason to prefix it either, it just hurts my eyes to see a global variable (even though it is static) without prefix.